modules/Surveys/Entry/Thanks.php

```diff
@@ -14,14 +20,9 @@
 </head>
 <body>
 <div class="container">
-    <div class="row">
-        <div class="col-md-offset-3 col-md-6">
-            <img height=100 src="survey_logo.jpg"/>
-        </div>
-    </div>
     <div class="row well">
         <div class="col-md-offset-2 col-md-8">
-            <h1><?= $survey->name ?></h1>
+            <h1><?php echo $surveyName; ?></h1>
```

Comment:
Question: The short hand tags "<?=" are available on php 5.4 and above. SuiteCRM 7.10 has a minimum requirement of 5.5.9. So why do we need to change these?

Reply:
PSR-1

Reply:
Yes my bad, that change is unnecessary

Reply:
Ah OK. I wanted to make sure that the community member who raised it, was using the correct php version.